src/ankit/Test_14_All_Test_TestNG_Conversion/Test13_ValidateEmployeeCountBasedOnDeptFromPropertyFile.java

```diff
@@ -0,0 +1,82 @@
+package ankit.Test_14_All_Test_TestNG_Conversion;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Properties;
+import java.util.Set;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import ankit.AM_base.PredefinedMethods;
+import ankit.AM_util.PropertyFileReader_util;
+
+public class Test13_ValidateEmployeeCountBasedOnDeptFromPropertyFile {
+
+	WebDriver driver ;
+
+	@BeforeClass
+	public void setUp(){
+		driver = PredefinedMethods.launchBrowser("http://www.automationbykrishna.com");
+
+	}
+	@AfterClass
+	public void tearDown(){
+		driver.close();
+	}
+
+	private HashMap<String, Integer> getEmployeeCoutAsPerDept(WebDriver driver) {
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method